src/NonUploader.tsx

```diff
@@ -0,0 +1,39 @@
+import compact from 'lodash/compact'
+import { fileToBase64 } from '@seasonedsoftware/utils'
+
```

Comment:
Let's look for a better name for this guy... If it doesn't upload, what does it actually do?

Reply:
Prepare the file to be uploaded in the same way it would be to Firebase. And do not upload to anywhere, just skip this part.